src/views/SystemRepositoryTable/index.tsx

```diff
@@ -126,56 +122,93 @@ export const SystemRepositoryTable = () => {
     return classnames(marginClassName, 'fa fa-caret-up');
   };
 
+  // As current CedarSystemBookMark type stands, db bookmarks need to be filted through existing CEDAR data get get the full bookmark.
+  const filterBookmarks = (
```

Comment:
Some nitpicks on coding style:
* This function definition can be moved outside the component, it doesn't depend on anything inside the component.
* `bookmarkData` isn't a great name for the parameter that's a list of systems, I think a better name would be `systems` or `allSystems` or similar.
* I'd probably separate the filtering out into a separate `.filter()` call.
* The returned `<BookmarkCard />` component needs a key prop.

To sum up, what I'd propose is this function definition, moved out of the component definition:
```
const filterBookmarks = (
  systems: SystemInfo[],
  savedBookMarks: CedarSystemBookMark[]
) =>
  systems
    .filter(system =>
      savedBookMarks.some(bookmark => bookmark.cedarSystemId === system.id)
    )
    .map(system => (
      <BookmarkCard type="systemList" key={system.id} {...system} />
    ));
```

Reply:
Great suggestion Dylan, thanks I'll incorporate these.